mlir/lib/Dialect/MIOpen/Transforms/AlignTiling.cpp

```diff
@@ -127,9 +127,6 @@ template <typename T> struct MILARewritePattern : public OpRewritePattern<T> {
     }
     nTWCopy->setAttr("bounds", b.getArrayAttr(twCopyBoundsAttr));
 
-    // 5. Adjust the copy to show the correct argument as global
-    nTWCopy->setAttr("globalArg", b.getIndexAttr(0));
```

Comment:
@sjw36 Need your comment on this proposed change. I'm not that familiar with the expected syntax in this pass.

Reply:
I added that line back when I reworked `threadwise_copy`, and so I figure I can remove it since the `threadwise_copy` rewrite will handle this without the attribute.